cell_classification/promix_naive.py

```diff
@@ -69,11 +72,15 @@ def train_step(model, optimizer, loss_fn, aug_fn, loss_mask, x, y_gt):
         Returns:
             tf.Tensor: loss value
         """
-        x_aug, loss_mask_aug, y_gt_aug = aug_fn(x, loss_mask, y_gt)
+        loss_mask = tf.expand_dims(tf.cast(loss_mask, tf.uint8), -1)
+
+        x_aug, cat = aug_fn(x, tf.concat([loss_mask, y_gt], axis=-1))
+        loss_mask_aug, y_gt_aug = tf.split(cat, [loss_mask.shape[-1], y_gt.shape[-1]], axis=-1)
```

Comment:
Couldn't follow this logic

Reply:
`aug_fn` applies non-spatial augmentations (noise, blur, contrast) and bilinear interpolation for the spatial augmentations only for the first argument, for the second argument nearest neighbor interpolation is used and no non-spatial augmentations. Since we want noise/blur/contrast only on the multiplexed image, we split and concat stuff around here..